salt/modules/virt.py

```diff
@@ -892,6 +993,10 @@ def _disk_profile(profile, hypervisor, disks=None, vm_name=None, image=None, poo
     elif hypervisor in ['qemu', 'kvm']:
         overlay = {'format': 'qcow2',
                    'model': 'virtio'}
+    elif hypervisor in ['bhyve']:
+        overlay = {'format': 'raw',
+                   'model': 'virtio',
+                   'sparse_volume': False}
```

Comment:
you'll need to change the function name in a separate commit since it doesn't create only qemu images ;)

Reply:
Change which function name?

Reply:
I didn't touch the contents of _qemu_image_create(). The code block referenced by your comment resides in _disk_profile()

Reply:
oh my bad... sorry for the inconvenience, it looks like I misread the patch ;)